src/main/java/net/fabricmc/installer/Handler.java

```diff
@@ -39,6 +41,7 @@
 	public JCheckBox snapshotCheckBox;
 
 	private JPanel pane;
+	private final ResourceBundle bundle = Utils.BUNDLE;
```

Comment:
I would get rid of this tbh, as in most places you are just calling Utils.BUNDLE directly.

Reply:
Yes, using that directly.